src/Components/Nav/NavBanner.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './NavBanner.scss';
+
+class NavBanner extends React.Component {
+  state = {
+    bannerChange: 0,
+  };
+
+  componentDidMount() {
+    setInterval(this.bannerChange, 1500);
+  }
+
+  bannerChange = () => {
+    const { bannerChange } = this.state;
+    if (bannerChange === 0) this.setState({ bannerChange: 1 });
+    if (bannerChange === 1) this.setState({ bannerChange: 0 });
+  };
```

Comment:
추후 두 종류가 아니라 세 종류 이상일 때 확장성을 고려하여 짜신 코드인가요?
아니라면 아래와 같이 수정할 수 있을 것 같습니다.
```suggestion
  bannerChange = () => {
    const { bannerChange } = this.state;
    this.setState({ bannerChange: bannerChange===0 ? 1 : 0 });
  };
```